includes/wizards/audience/class-audience-subscriptions.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Audience Subscriptions Wizard
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Audience Subscriptions Wizard.
+ */
+class Audience_Subscriptions extends Wizard {
+	/**
+	 * Admin page slug.
+	 *
+	 * @var string
+	 */
+	protected $slug = 'newspack-audience-subscriptions';
+
+	/**
+	 * Parent slug.
+	 *
+	 * @var string
+	 */
+	protected $parent_slug = 'newspack-audience';
+
+
+	/**
+	 * Get the name for this wizard.
+	 *
+	 * @return string The wizard name.
+	 */
+	public function get_name() {
+		return esc_html__( 'Audience Development / Subscriptions', 'newspack-plugin' );
+	}
+
+	/**
+	 * Add Subscriptions page.
+	 */
+	public function add_page() {
+		add_submenu_page(
+			$this->parent_slug,
+			$this->get_name(),
+			esc_html__( 'Subscriptions', 'newspack-plugin' ),
+			$this->capability,
+			$this->slug,
+			[ $this, 'render_wizard' ]
+		);
+	}
+
+	/**
+	 * Enqueue scripts and styles.
+	 */
+	public function enqueue_scripts_and_styles() {
+		if ( ! $this->is_wizard_page() ) {
+			return;
+		}
+
+		parent::enqueue_scripts_and_styles();
+		wp_enqueue_script( 'newspack-wizards' );
+		wp_localize_script(
+			'newspack-wizards',
+			'newspackAudienceSubscriptions',
+			[
+				'tabs' => [
+					[
+						'path'        => '/configuration',
+						'title'       => esc_html__( 'Configuration', 'newspack-plugin' ),
+						'header'      => esc_html__( 'Manage Subscriptions settings in Woo Memberships', 'newspack-plugin' ),
+						'description' => esc_html__( 'You can manage the details of your subscription offerings in the Woo Memberships plugin.', 'newspack-plugin' ),
+						'href'        => admin_url( 'admin.php?page=wc-settings&tab=memberships' ),
+						'btn_text'    => esc_html__( 'Manage Subscriptions', 'newspack-plugin' ),
+					],
+					[
+						'path'        => '/revenue',
+						'title'       => esc_html__( 'Revenue', 'newspack-plugin' ),
+						'header'      => esc_html__( 'View Subscription Revenue in WooCommerce', 'newspack-plugin' ),
+						'description' => esc_html__( 'You can view revenue data from Donations and Subscriptions in the WooCommerce Plugin.', 'newspack-plugin' ),
+						'href'        => admin_url( 'admin.php?page=wc-reports' ),
+						'btn_text'    => esc_html__( 'View Subscription Revenue', 'newspack-plugin' ),
+					],
```

Comment:
During the presentation, it was pointed out that we shouldn't render this tab while we don't have the link for our custom revenue report.

We can preserve the code but comment it out with a docblock stating the reason.

Reply:
Refactored in 60a1787661a8d79fd9b5476319d7ef249c052d35. PHPCS required the space between the block comments, looks little weird.